### PR TITLE
ci(plasma-*): Bump node.js to 14.21.2

### PR DESCRIPTION
**Pull request by Yakutoc**
Для того что бы корректно обновить версию Storybook, нужно поднять версию **Node.js**

Пока не удалось понять почему при версии **Node.js 14.0.0** команды:

`npx storybook@6.5.16 upgrade` или `npx storybook upgrade` 

не обновляют сам Storybook и его зависимости, что отраженны в **package-lock.json**  и **package.json**.

Так же в рамках этой задачи переехали на `actions/setup-node@v3` для установки Node.js и что бы у нас была возможность использовать `.nvmrc` в настройках.

```
      - name: Use Node.js 14.x
        uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'
```  

    

:baby_chick: Download canary assets:

[colors--canary.332.4170612742.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.332.4170612742.xml)  
[color_metro_ios-swift--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.332.4170612742.swift)  
[color_metro_kotlin--canary.332.4170612742.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.332.4170612742.kt)  
[color_metro_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.332.4170612742.ts)  
[color_sbermarket_ios-swift--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.332.4170612742.swift)  
[color_sbermarket_kotlin--canary.332.4170612742.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.332.4170612742.kt)  
[color_sbermarket_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.332.4170612742.ts)  
[color_sberprime_ios-swift--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.332.4170612742.swift)  
[color_sberprime_kotlin--canary.332.4170612742.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.332.4170612742.kt)  
[color_sberprime_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.332.4170612742.ts)  
[color_selgros_ios-swift--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.332.4170612742.swift)  
[color_selgros_kotlin--canary.332.4170612742.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.332.4170612742.kt)  
[color_selgros_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.332.4170612742.ts)  
[color_smbusiness_ios-swift--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.332.4170612742.swift)  
[color_smbusiness_kotlin--canary.332.4170612742.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.332.4170612742.kt)  
[color_smbusiness_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.332.4170612742.ts)  
[PlasmaTokensColor--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.332.4170612742.swift)  
[shadow_sbermarket_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.332.4170612742.ts)  
[typo_mage_ios-swift--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.332.4170612742.swift)  
[typo_mage_kotlin--canary.332.4170612742.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.332.4170612742.kt)  
[typo_mage_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.332.4170612742.ts)  
[typo_plasma_ios-swift--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.332.4170612742.swift)  
[typo_plasma_kotlin--canary.332.4170612742.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.332.4170612742.kt)  
[typo_plasma_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.332.4170612742.ts)  
[typo_ruler_ios-swift--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.332.4170612742.swift)  
[typo_ruler_kotlin--canary.332.4170612742.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.332.4170612742.kt)  
[typo_ruler_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.332.4170612742.ts)  
[typo_sage_ios-swift--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.332.4170612742.swift)  
[typo_sage_kotlin--canary.332.4170612742.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.332.4170612742.kt)  
[typo_sage_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.332.4170612742.ts)  
[typo_sbermarket_ios-swift--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.332.4170612742.swift)  
[typo_sbermarket_kotlin--canary.332.4170612742.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.332.4170612742.kt)  
[typo_sbermarket_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.332.4170612742.ts)  
[typo_soulmate_ios-swift--canary.332.4170612742.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.332.4170612742.swift)  
[typo_soulmate_kotlin--canary.332.4170612742.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.332.4170612742.kt)  
[typo_soulmate_react-native--canary.332.4170612742.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.332.4170612742.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-temple@1.131.2-canary.332.4170612742.0
  npm install @salutejs/plasma-ui@1.164.1-canary.332.4170612742.0
  # or 
  yarn add @salutejs/plasma-temple@1.131.2-canary.332.4170612742.0
  yarn add @salutejs/plasma-ui@1.164.1-canary.332.4170612742.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-332/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-332/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-332/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-332/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-332/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-332/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-332/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-332/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-332/

**Comment by Yakutoc** [comment]
@Yeti-or @neretin-trike  Предлагаю использовать такой синтаксис для наших actions

```
       with:
            node-version-file: '.nvmrc'
```

[node-version-file](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file) 

**Мотивация:**  

- У нас будет одна точка управления зависимостью. 
- И мы точно будем уверены что у нас и локально и CI одна версия Node.js   

Что думаете?

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-332/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-332/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-332/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-332/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-332/temple-storybook/

**Review by neretin-trike — APPROVED** [review]

File: .github/workflows/versionate-docs.yml [hunk 1]
```diff
@@ -21,10 +21,10 @@ jobs:
         # https://github.com/intuit/auto/issues/1030
         run: git config --local --unset http.https://github.com/.extraheader
 
-      - name: Use Node.js 14
+      - name: Use Node.js 14.x
         uses: actions/setup-node@v1
         with:
-          node-version: 14
+          node-а: 14.x
```
Comment (Yeti-or) [inline .github/workflows/versionate-docs.yml:27]: node-a ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ git commit -m "fix(plasma-web): Fix component Y"
 
 ## Node, npm versions
 
-Для корректной работы, необходимо использовать `node` версии `v14`, а также `npm` версии `6.x`. Или же воспользоваться командой `nvm use` в корне репозитория.
+Для корректной работы, необходимо использовать `node` версии `v14.x`, а также `npm` версии `6.x`. Или же воспользоваться командой `nvm use` в корне репозитория.
```
Comment (Yeti-or) [inline CONTRIBUTING.md:34]: я вот только не уверен что нам подойдут все версии node

пример: https://github.com/salute-developers/plasma/pull/323#discussion_r1102503768

Comment (Yeti-or) [reply CONTRIBUTING.md:34]: но в целом не существенно я думаю, давай так оставим

**Comment by Yeti-or** [comment]
Давай вообщем на файл поменяем

File: .github/workflows/versionate-docs.yml (hunk 1 above)
Comment (Yakutoc) [reply .github/workflows/versionate-docs.yml]: @Yeti-or Да, заметил, копипаста подвела. Уже поправил.

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-332/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-332/

File: .github/workflows/cypress-ct.yml [hunk 3]
```diff
@@ -11,10 +11,10 @@ jobs:
     steps: 
     - uses: actions/checkout@v2
 
-    - name: Use Node.js 14
-      uses: actions/setup-node@v1
+    - name: Use Node.js 14.x
+      uses: actions/setup-node@v3
       with:
-        node-version: 14
+        node-version-file: '.nvmrc'
```
Comment (Yakutoc) [inline .github/workflows/cypress-ct.yml:17]: @Yeti-or Уточню, что мы друг друга поняли правильно )

Используем .nvmrc?

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-332/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-332/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-332/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-332/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-332/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-332/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-332/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-332/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-332/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-332/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-332/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-332/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-332/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-332/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-332/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-332/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-332/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-332/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-332/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-332/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-332/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **💀 Missing report, something went wrong**

Check out current report in job artifacts!

File: .github/workflows/cypress-ct.yml (hunk 3 above)
Comment (Yeti-or) [reply .github/workflows/cypress-ct.yml:17]: Да

File: .github/workflows/cypress-ct.yml [hunk 4]
```diff
@@ -11,10 +11,10 @@ jobs:
     steps: 
     - uses: actions/checkout@v2
 
-    - name: Use Node.js 14
-      uses: actions/setup-node@v1
+    - name: Use Node.js 14.x
```
Comment (Yeti-or) [inline .github/workflows/cypress-ct.yml:14]: Кажется 14 лишняя инфа

**Review by Yeti-or — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-332/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-332/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-332/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-332/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-332/temple-storybook/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-332/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-332/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-332/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-332/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-332/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-332/

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/pr/demo-canvas-app-pr-332/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-332/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-332/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-332/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-332/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-332/temple-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Pull request merged**